### PR TITLE
Fix file stream leaks

### PR DESCRIPTION
**Pull request by kitwtnb**
## Issue
- none

## Overview (Required)
- File stream leaks when exception occurs therefore close method moved to finally block.

## Links
- none

### CODE REVIEW

File: app/src/main/java/io/github/droidkaigi/confsched2017/util/AssetsUtil.java [hunk 1]
```diff
@@ -23,15 +23,23 @@ private AssetsUtil() {
...
+            is = context.getAssets().open("json/" + jsonFileName);
             int size = is.available();
             byte[] buffer = new byte[size];
             is.read(buffer);
-            is.close();
             json = new String(buffer, "UTF-8");
         } catch (IOException e) {
             Timber.e(AssetsUtil.class.getSimpleName(), e.getMessage(), e);
+
+            try {
```
Comment (sys1yagi) [inline app/src/main/java/io/github/droidkaigi/confsched2017/util/AssetsUtil.java]: It seems missing "finally" block. 🤔

File: app/src/main/java/io/github/droidkaigi/confsched2017/util/AssetsUtil.java [hunk 2]
```diff
@@ -23,15 +23,23 @@ private AssetsUtil() {
...
+                if (is != null) {
+                    is.close();
+                }
+            } catch (IOException exp) {
+                Timber.e(AssetsUtil.class.getSimpleName(), e.getMessage(), exp);
+            }
         }
```
Comment (jmatsu) [inline app/src/main/java/io/github/droidkaigi/confsched2017/util/AssetsUtil.java]: input stream is not closed when succeeded.
As @sys1yagi said, closing stream in `finally` block works well in this case.

**Comment by kitwtnb** [comment]
I was wrong. 😥 
I fix it.
Thank you for @sys1yagi and @jmatsu ! 😄

**Comment by konifar** [comment]
@kitwtnb Sorry, conflict occurred. Can you merge master? 🙇

**Comment by konifar** [comment]
LGTM! 💯

**Comment by kitwtnb** [comment]
OK, I do it later!

**Comment by konifar** [comment]
LGTM! Thanks 👍

**Pull request merged**